README.md

```diff
@@ -39,9 +39,6 @@
       <a href="https://github.com/psf/black">
       <img src="https://img.shields.io/badge/code%20style-black-000000.svg" alt="Cod Style">
       </a>
-      <a href="https://travis-ci.com/DeepRegNet/DeepReg">
```

Comment:
Is there any other way we can display build status now? I think this is important to have on the README.

Reply:
Agree

Reply:
Maybe but I will do that later, as I didn't find out a good way to do so for now.

Reply:
@mathpluscode see:

https://andresand.medium.com/adding-github-action-workflow-status-badge-to-your-repository-22ccea025af6

or

https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge